content/docs/2.10/scalers/prometheus.md

```diff
@@ -25,7 +25,14 @@ triggers:
     cortexOrgID: my-org # DEPRECATED: This parameter is deprecated as of KEDA v2.10 in favor of customHeaders and will be removed in version 2.12. Use custom headers instead to set X-Scope-OrgID header for Cortex. (see below)
     customHeaders: X-Client-Id=cid,X-Tenant-Id=tid,X-Organization-Id=oid # Optional. Custom headers to include in query. In case of auth header, use the custom authentication or relevant authModes.
     ignoreNullValues: false # Default is `true`, which means ignoring the empty value list from Prometheus. Set to `false` the scaler will return error when Prometheus target is lost
-    unsafeSsl: "false" #  Default is `false`, Used for skipping certificate check when having self signed certs for Prometheus endpoint
+    unsafeSsl: "false" #  Default is `false`, Used for skipping certificate check when having self signed certs for Prometheus endpoint    
+    # Required when using Azure managed service for Prometheus
+    authenticationRef:
+    # Valid when using Azure managed service for Prometheus
+    cloud: Private # Default is `AzurePublicCloud`
+    # Required when cloud = Private
+    azureManagedPrometheusResourceURL: https://prometheus.monitor.azure.airgap/.default
+
```

Comment:
Let's leave this out and cover it in the details
```suggestion
    # Valid when using Azure managed service for Prometheus
    cloud: Private # Default is `AzurePublicCloud`
    # Required when cloud = Private
    azureManagedPrometheusResourceURL: https://prometheus.monitor.azure.airgap/.default
```

Reply:
done